### ISSUE TITLE
Quickly switching from commit results to graph

### ISSUE THREAD

**Issue by Kha** [IPDSnelting/velcom#35]
For example, in codespeed every table row links to the respective graph

**Comment by oudemia** [comment]
The respective graph being which one? I suppose showing the detail graph of the corresponding repo with only the metric selected that was clicked on in the commit detail view would be nice, but which time frame should be shown? 

Only a short period like a week with the selected commit in the centre? That could be nice to visualize the significance of a commit. Or maybe the entire graph to give a broad overview of the perfomance?

The picture attached shows a highlighted commit in a graph with 800 commits selected, which in my opinion already gives a quite good understanding of the change in performance.

![grafik](https://user-images.githubusercontent.com/52839122/89016860-b86cc280-d319-11ea-883f-623bf6012501.png)

**Comment by Kha** [comment]
> the detail graph of the corresponding repo with only the metric selected that was clicked on in the commit detail view

Yes.

> but which time frame should be shown?

Good question; codespeed might not even show the commit in question so we can only do better here... I think a week of context is a reasonable default for repositories of all sizes. Would be great if one could quickly zoom out from there, but that should probably go in a different issue.

**Comment by oudemia** [comment]
Sounds good. Should the selected commit be centred o the newest commit shown?

**Comment by Kha** [comment]
I'd say centered makes sense, to have some context on both sides. In many cases the selected commit will be relatively new though, which I assume would automatically degrade gracefully to a (mostly) right-aligned view.

**Comment by I-Al-Istannen** [comment]
Clicking the row sounds a bit problematic, as you can no longer copy test and it isn't easy to discover. If we sacrifice some horizontal space we could add a new column:
![image](https://user-images.githubusercontent.com/20284688/94940634-0b0e5a80-04d4-11eb-9d2e-e41928bfa95f.png)

We could also implement clicking/double clicking a row, if you prefer that.

**Comment by Kha** [comment]
Making the row clickable should not interfere with selecting text by click-dragging; also compare with codespeed, where it's done exactly that way.

**Comment by Kha** [comment]
By the way, compared to codespeed Velcom is *very* generous with table spacing. A more compact layout would make it easier to visually scan the table.

**Comment by I-Al-Istannen** [comment]
Ah, I normally double/triple click text if I know I want a word/line. It also doesn't work on the codespeed page I tried it on (http://speedcenter.informatik.kit.edu/compprakt/changes/?rev=2315e85972e63ea327c4d115ffe623253b520440&exe=42&env=i44pc15):
![output](https://user-images.githubusercontent.com/20284688/94949088-a6a5c800-04e0-11eb-87ee-836ff2f52bc2.gif)

It directly navigates away when I let go of my mouse. I think that's fixable though, as I tried something similar a while ago. If it doesn't work, text selection is a bit broken but that's probably not the end of the world.

Yea, the table can be condensed a bit. We were going over the UI a while ago trimming margins and that's a good candidate as well. The material design framework is quite keen on whitespace by default.
